README.md

```diff
@@ -800,6 +800,10 @@ export JAEGER_TRACE=0.0.0.0:6831
 # any buildctl command should be traced to http://127.0.0.1:16686/
 ```
 
+> On Windows, once you have the [`jaeger-all-in-one.exe`](https://www.jaegertracing.io/docs/1.57/getting-started/#all-in-one) running,
```

Comment:
This implies as if you can't run linux containers (and above docker run) on windows. Smth. like "if you are running Jaeger outside of container" ?

Reply:
that's right, will fix. Also just noticed that `OTEL_TRACES_EXPORTER` is too global and might be used by other apps like for instance `registry.exe` broke coz of this. Will stick to `JAEGER_TRACE=0.0.0.0:6831`.